nntrainer/tensor/tensor.cpp

```diff
@@ -560,7 +560,7 @@ Tensor Tensor::getSharedDataTensor(const TensorDim dim_, unsigned int offset,
   Tensor ret = *this;
   ret.dim = dim_;
 
-  if (ret.bytes() + offset > bytes())
+  if (dim_.getDataLen() + offset > dim.getDataLen())
```

Comment:
Not related to this PR, i recommend to update map to use getDatLen as well, it is confusing to have one funciton use bytes and other use length.